docs/source/tutorial/visual_representations.rst

```diff
@@ -0,0 +1,3 @@
+Visual Representations
+======================
+.. todo:: @mberr
```

Comment:
please add an example of using this (and if there's some nice images, that would be great too! put them in `/docs/source/img/`)

Reply:
you crazy dude

Reply:
removed this file in https://github.com/pykeen/pykeen/pull/965/commits/dbd240149ededb3b628da055f6c006a2297e6cbb